.aws/graasp-dev.json

```diff
@@ -130,9 +130,9 @@
       "entryPoint": null,
       "portMappings": [
         {
-          "hostPort": 8080,
+          "hostPort": 8081,
```

Comment:
Need to check with @spaenleh why this is needed. Also eventually should check how to apply to staging and production.

Reply:
This is the container that hosts the nudenet validator (inappropriate/nudity) It is used when publishing content to the library. If the library is not used you can safely remove the whole container definition. In the current state, Ia m not sure it would work (the internal port should not be changed, only the the exposed post I think).